jiant/evaluate.py

```diff
@@ -101,11 +101,14 @@ def evaluate(
             with torch.no_grad():
                 if isinstance(cuda_device, int):
                     batch = move_to_device(batch, cuda_device)
-                out = model.forward(task, batch, predict=True)
+                out = model.forward(task=task, batch=batch, predict=True)
+            if task is not None:
+                task.update_metrics(out, batch)
```

Comment:
One more subtle issue: in `/models.py` we missed a call to an [update metrics method](https://github.com/nyu-mll/jiant/blob/41edb47250d37c278f87c58cd53e7a31378ba905/jiant/models.py#L955). It's easy to overlook because it has an unusual name, "`update_diagnostic_metrics`".